plugins/janus_audiobridge.c

```diff
@@ -3931,8 +4039,8 @@ static void *janus_audiobridge_handler(void *data) {
 			answer->s_name = g_strdup(s_name);
 			/* Add a fmtp attribute */
 			janus_sdp_attribute *a = janus_sdp_attribute_create("fmtp",
-				"%d maxplaybackrate=%"SCNu32"; stereo=0; sprop-stereo=0; useinbandfec=0\r\n",
-					participant->opus_pt, participant->room->sampling_rate);
+				"%d maxplaybackrate=%"SCNu32"; stereo=0; sprop-stereo=0; useinbandfec=%s\r\n",
+					participant->opus_pt, participant->room->sampling_rate, use_fec ? "1" : "0");
```

Comment:
Why a string? It can be %d with `use_fec ? 1 : 0`.

Reply:
Ok.

Thank you for your review of the code. I just would like to know if you have tested the FEC and verified that it improve audio quality. And I would like to know if you have found some interest in the implementation.

I’m on holiday since 14 may, so sorry for the lack of feedback during that time.

Reply:
I had no time to test anything yet, sorry. It's definitely of interest, but of course so many changes in something that currently works require a deep review, which is why I'm nitpicking pretty much everything... Thanks again and enjoy your vacation! I'll be abroad for a conference on the 14th, so I'll get back to you a couple of days later probably.